src/components/AddNewOpinion/AddNewOpinion.interface.ts

```diff
@@ -0,0 +1,7 @@
+export default interface IAddNewOpinion {
+    // username: string;
```

Comment:
komentarz do usunięcia?

Reply:
Tą linijkę myślałem odkomentować. Bo do tej pory opinie były przypisywane do jakiegoś userID, które się wyświetlało w górnej części opinii. Był to nic nie mówiący numer. Przy dodaniu username opinie miały przypisane imiona userów, którzy je utworzyli :) tylko miałem tam coś niedokończone, więc poszło w komentarz na później ;)